packages/types/src/interfaces/scaleInfo/definitions.ts

```diff
@@ -41,7 +41,7 @@ export default {
       variants: 'Vec<SiVariant>'
     },
     SiTypeDefPrimitive: {
-      _enum: ['Bool', 'Char', 'Str', 'U8', 'U16', 'U32', 'U64', 'U128', 'I8', 'I16', 'I32', 'I64', 'I128']
+      _enum: ['Bool', 'Char', 'Str', 'U8', 'U16', 'U32', 'U64', 'U128', 'U256', 'I8', 'I16', 'I32', 'I64', 'I128', 'I256']
```

Comment:
Yes, I was actually wondering about this - I'm assuming the same changes are made on the scale-info side?

Reply:
If you mean https://crates.io/crates/scale-info then yes, @Robbepop said so.

Unfortunately there are bunch of changes needed to that crate for solang, but that's another matter.

Reply:
The above needs to match with https://github.com/paritytech/scale-info/blob/master/src/ty/mod.rs#L166-L193

Reply:
So yes, I'll make a PR for scale-info